client/js/components/procedure/DpSimplifiedNewStatementForm.vue

```diff
@@ -492,7 +492,7 @@ export default {
       isLoading: false,
       isSaving: false,
       values: {
-        authoredDate: '',
+        authoredDate: this.initialAuthoredDate,
```

Comment:
Are you sure this can be used here? EsLint says: "The computed property cannot be used in `data()` because it is before initialization.(vue/no-computed-properties-in-data)"

Reply:
Ah I see, true we should not be doing it like that. I just wanted to avoid a ternary in the data props and use a computed instead but wasn't aware that my implementation could be prone to errors. Done here https://github.com/demos-europe/demosplan-core/pull/1365/commits/fc71eb1c2b9e9434d39864247e4a11c9ebac1f40